apps/client/src/features/menu/MenuBar.tsx

```diff
@@ -128,12 +167,14 @@ const MenuBar = (props: MenuBarProps) => {
         {...buttonStyle}
         icon={<IoSaveOutline />}
         isDisabled={appMode === AppMode.Run}
-        clickHandler={downloadRundown}
+        clickHandler={() => setIsModalOpen(true)}
         tooltip='Export project file'
         aria-label='Export project file'
         size='sm'
       />
 
+      {exportOptionsModal()}
```

Comment:
Few code style nitpicks:

is there a reason for this to be a function and not a react component?
In this case, I think it might just work fine, but I believe it to be a bad pattern to follow

we generally tend to use the `function Component` style rather than arrow function, it is nice to be consistent. I would have generally not mentioned it, but either way, all components must start with Uppercase, I didn't even know that react would let you name it otherwise

we will likely want to share this component so we can extract it in its own file to the `common/components` directory